api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -26,21 +27,22 @@ public void setup() {
   }
 
   @Test
-  public void testDummyUsernameIsNotTaken() throws IOException {
+  public void testDummyUsernameIsNotTaken() {
     assertThat(service.isUsernameTaken("username-that-should-not-exist")).isFalse();
   }
 
   @Test
-  public void testDirectoryServiceUsernameIsTaken() throws IOException {
+  public void testDirectoryServiceUsernameIsTaken() {
     assertThat(service.isUsernameTaken("directory-service")).isTrue();
   }
 
   @Test
-  public void testCreateAndDeleteTestUser() throws IOException {
-    service.createUser("Integration", "Test", "integration.test", "notasecret");
-    assertThat(service.isUsernameTaken("integration.test")).isTrue();
-    service.deleteUser("integration.test");
-    assertThat(service.isUsernameTaken("integration.test")).isFalse();
+  public void testCreateAndDeleteTestUser() {
+    String userName = String.format("integration.test.%d", Clock.systemUTC().millis());
```

Comment:
So the only downside here is that after a while if there are failures we may wind up with not-cleaned-up accounts in Gsuite and hit our quota limit. I guess we'll just delete them if this happens.

Reply:
Yep, true. I think this is the lesser of the two evils though. Hopefully this doesn't flake often.